src/systems/air_speed/AirSpeed.cc

```diff
@@ -307,4 +307,4 @@ GZ_ADD_PLUGIN(AirSpeed, System,
 GZ_ADD_PLUGIN_ALIAS(AirSpeed, "gz::sim::systems::AirSpeed")
 
 // TODO(CH3): Deprecated, remove on version 8
-GZ_ADD_PLUGIN_ALIAS(AirSpeed, "ignition::gazebo::systems::AirSpeed")
+GZ_ADD_PLUGIN_ALIAS(AirSpeed, "gz::sim::systems::AirSpeed")
```

Comment:
This just needs to be removed.

Reply:
Removed in [fa23fcc](https://github.com/gazebosim/gz-sim/pull/2055/commits/fa23fcce063854af27c64421fb4d980be169514a)